source/_static/_templates/theme/layout.html

```diff
@@ -0,0 +1,28 @@
+{# Import the theme's layout. #}
+{% extends "!layout.html" %}
+
+{# Adds Google Analytics events on button click. #}
+{%- block extrahead %}
+<script>
+var _gaq = _gaq || [];
+_gaq.push(['_setAccount', 'UA-343802-3']);
+_gaq.push(['_trackPageview']);
+</script>
+{{ super() }}
+{%- endblock %}
+
+
+{%- block relbar2 %}
+<div class="md-relbar2__inner md-grid">
+<div class="md-content">
+  <article class="md-content__inner md-typeset" role="main">    
+    <h4>Contact Us </h4>
+       <p>For free technical help, visit the Percona <a class="reference external" href="https://forums.percona.com/c/mongodb/percona-server-for-mongodb/17?utm_campaign=Doc%20pages" target="_blank">Community Forum</a>.<br>
+        <p>To report bugs or submit feature requests, open a <a class="reference external" href="https://jira.percona.com/projects/PSMDB/issues/" target="_blank">JIRA</a> ticket.<br>
+         <p>For paid <a class="reference external" href="https://www.percona.com/services/support"> support </a> and <a class="reference external" href="https://www.percona.com/services/managed-services"> managed </a> or <a class="reference external" href="https://www.percona.com/services/consulting">consulting services </a>, contact <a class="reference external" href="https://www.percona.com/about-percona/contact" target="_blank">Percona Sales</a>.</p>
```

Comment:
Could this part be separated and included here instead? This may make it easier to share it across projects.

Reply:
What part are you talking about?

Reply:
Well, the exact block I marked (Line 19-22, the "Contact us" block).